specification/containerregistry/resource-manager/readme.md

```diff
@@ -26,14 +26,23 @@ These are the global settings for the ContainerRegistry API.
 
 ``` yaml
 openapi-type: arm
-tag: package-2023-08-preview
+tag: package-preview-2023-11
```

Comment:
@raymondluo1 please keep consistent with the previous naming. So we tag this `package-2023-11-preview`